app/views/idv/in_person/encrypt.html.erb

```diff
@@ -0,0 +1,31 @@
+<% title t('idv.titles.review') %>
+
+<h5 class="my1 caps bold accent-blue">
+  <%= t('in_person_proofing.step', step: 2) %>
+</h5>
+
+<br/>
+
+<h1 class="h3">
+  Re-enter your login.gov password to protect your information
```

Comment:
I think we have a key for this? `idv.titles.session.review` as long as we're here